proto/powergate/user/v1/user.proto

```diff
@@ -503,6 +505,13 @@ message StorageDealRecord {
   int64 time = 3;
   bool pending = 4;
   StorageDealInfo deal_info = 5;
+  int64 transfer_size = 6;
+  int64 data_transfer_start = 7;
+  int64 data_transfer_end = 8;
+  int64 sealing_start = 9;
+  int64 sealing_end = 10;
+  string err_msg = 11;
+  int64 updated_at = 12;
```

Comment:
This one I might push a bit to change before merging. I've convinced myself that sending timestamp data in Protobuf as int isn't ideal. In order to use it properly, the consumer has to know about or make assumptions about the server implementation. This is probably nano seconds, but is it mill? Is it just seconds? Depending on the answer the client needs to process data differently. Some languages handle milliseconds nicely, while others deal with nanoseconds nicely. Anyway, long story short, I think its better to use `import "google/protobuf/timestamp.proto";` and `google.protobuf.Timestamp` to make this most flexible and deterministic on the client side. 

I think there are still times where sending some integer representation of time, paging apis being one of them. But in that case, the integer should be sent as part of the response purely as a token that is sent back with the next request, no meaning other than that, no need to parse or interpret the integer.

Reply:
Yep, sounds good to me. If that datatype is compatible with clients in different languages, sounds like a better option. I guess that's the case, so I'll change it.

Reply:
Extra point about this: I'll change the new time-related fields to this new better type. The `Time` field I guess you created it as `int64` before this knowing about his, so I'd keep that untouched since that would be a breaking change and this PR is backward -compatible.

We should remember changing all the timestamps to this new type when landing some PR that already justifies breaking the APIs (i.e: v3)